WalletWasabi.Documentation/CodeCoverage.md

```diff
@@ -1,25 +1,17 @@
 # Code coverage (How to)
 
-Wasabi Wallet is built using dotnet core. Given there is no cross-platform Profiling API like the one available on Windows, we
-use [AltCover](https://github.com/SteveGilham/altcover) package for instrumenting the assemblies and recording the execution
-coverage.
+Wasabi Wallet is built using dotnet core. Currently dotnet XUnit and MSTest templates come with support for coverlet. You can read more about this here: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-code-coverage?tabs=linux and also here https://codeburst.io/code-coverage-in-net-core-projects-c3d6536fd7d7. 
 
-So, first of all we need to install the package in the `WalletWasabi.Tests` project as follow:
+dotnet test --filter "FullyQualifiedName~WalletWasabi.Tests.UnitTests.Crypto" --collect:"XPlat Code Coverage"
 
-```sh
-dotnet add WalletWasabi.Tests/WalletWasabi.Tests.csproj package AltCover
-```
-
-
-Next, run:
-
-```sh
-dotnet test /p:AltCover=true /p:AltCoverLcovReport=lcov.info
-```
-
-As a result we get a `lcov.info` file containing the covered lines. In order to be able to see what lines
+As a result we get a `coverage.cobertura.info` file containing the covered lines. In order to be able to see what lines
 are covered we need to install a `vscode extension` called  [Coverage Gutters](https://github.com/ryanluker/vscode-coverage-gutters).
 
 Run vscode and click on "Watch":
 
 ![](https://i.imgur.com/W4hXXda.png)
+
+
+## Important
+
+Coverlet cannot instrument the binaries correctly because of the `PathMap` entries in the project files (*.csproj) and for this reason the easiest way to to fix it is by removing the entries. 
```

Comment:
what does this mean for coverage reports in pipeline builds?

Reply:
This means that we need to disable the PathMap or let coverlet know about it (I didn't find how).